src/updater/updater.cpp

```diff
@@ -44,6 +46,25 @@
 #include <tuple>
 #include <vector>
 
+namespace std
+{
+template <typename T1, typename T2, typename T3> struct hash<std::tuple<T1, T2, T3>>
+{
+    size_t operator()(const std::tuple<T1, T2, T3> &t) const
+    {
+        return hash_val(std::get<0>(t), std::get<1>(t), std::get<2>(t));
+    }
+};
+
+template <typename T1, typename T2> struct hash<std::tuple<T1, T2>>
+{
+    size_t operator()(const std::tuple<T1, T2> &t) const
+    {
+        return hash_val(std::get<0>(t), std::get<1>(t));
+    }
+};
```

Comment:
Comes from our own implementation of `std_hash`. Boost::hash implementation for tuples doesn't work across all the systems we test on https://github.com/Project-OSRM/osrm-backend/issues/3962#issuecomment-297096071